core/domain/story_services.py

```diff
@@ -187,33 +189,35 @@ def apply_change_list(story_id, change_list):
         raise
 
 
-def _save_story(committer_id, story, commit_message, change_list):
-    """Validates a story and commits it to persistent storage. If
-    successful, increments the version number of the incoming story domain
-    object by 1.
+def validate_explorations(exp_ids, raise_error):
+    """Validates the explorations in the given story and checks whether they
```

Comment:
`validate_exploration` seems like it will validate given explorations, maybe we should rename this to `validate_exp_ids_for_story` or something similar?

Reply:
But we are validating the explorations themselves corresponding to the given IDs, not the IDs themselves. How about `validate_explorations_for_story`?

Reply:
I agree let'smake it `validate_explorations_for_story`, thanks!

Reply:
Done